src/main/java/ChatBot.java

```diff
@@ -0,0 +1,53 @@
+import esme.Ui;
```

Comment:
Nicely done! I like how you separate your files in a clear and reader-friendly manner